server/viame_server/viame_detection.py

```diff
@@ -67,6 +69,19 @@ def _get_clip_meta(self, folder):
             'videoUrl': videoUrl,
         }
 
+    def _load_detections(self, folder):
+        detectionItems = list(
+            Item().findWithPermissions(
+                {"meta.detection": str(folder["_id"])}, user=self.getCurrentUser(),
+            )
+        )
+        detectionItems.sort(key=lambda d: d["created"], reverse=True)
+        if not len(detectionItems):
+            return None
+        file = Item().childFiles(detectionItems[0])[0]
+
+        return file
```

Comment:
Added helper to find the girder file containing the detection bytes.  Loading/parsing is handled elsewhere.